Different fixes for word usage examples

irina060981 opened the pull request:
1. For the https://github.com/alpheios-project/components/issues/610
changed sorting option

2. For the https://github.com/alpheios-project/components/issues/611
changed filter -> limit

balmas commented on src/locales/en-us/messages.json:
```diff
@@ -600,17 +600,17 @@
   },
   "WORDUSAGE_FILTERS_TEXTWORK_CLEAR": {
-    "message": "Clear textwork filter",
+    "message": "Clear textwork limit",
```
please change "textwork" to "work" here too. thanks!

irina060981 replied:
Oh, missed it - fixed

balmas commented on src/locales/en-us/messages-word-list.json:
```diff
@@ -95,17 +95,17 @@
     },
     "WORDLIST_FILTER_BY": {
-        "message": "Filter by",
+        "message": "Limit by",
```
I think it should be "Limit to"

irina060981 replied:
Fixed it - sorry for my poor english!

balmas replied:
Either is grammatically correct :-)

balmas commented on src/locales/en-us/messages-word-list.json:
```diff
@@ -95,17 +95,17 @@
     },
     "WORDLIST_FILTER_CLEAR": {
-        "message": "Clear filter by",
+        "message": "Clear limit by",
```
can we say "Clear Limits"

irina060981 replied:
Fixed

irina060981 commented:
Also I was going to add changes for word usage get result worflow by the following issues
https://github.com/alpheios-project/components/issues/619
https://github.com/alpheios-project/components/issues/620

I have removed Get results and Get results without limits because it doesn't make any sense without Get results button.
I have added auto filter on changing filtering properties.

Hope it looks better!

irina060981 commented:
> looks ok, except I'm not sure about the removal of the collapsed header. We still want the ability to show or hide the filters (don't we?)

I didn't remove the collapsed header? I only put all filtering and sorting staff into it - like this:
First open

![image](https://user-images.githubusercontent.com/12377640/59804404-9096e700-9331-11e9-85c5-cb4bb34974fd.png)

click show limits

![image](https://user-images.githubusercontent.com/12377640/59804415-9d1b3f80-9331-11e9-8872-c03a76018814.png)

select author in filter
![image](https://user-images.githubusercontent.com/12377640/59804450-b1f7d300-9331-11e9-86e6-162d288ee630.png)

irina060981 commented:
As everything else is reviewed and has no changes requested - I will go ahead and merge it!
